CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+### Fixed
```

Comment:
Thanks for your contribution!

I don't believe this goes in the CHANGELOG though as it's not a user-visible change.

Reply:
Oh and please add yourself to AUTHORS